pom.xml

```diff
@@ -21,6 +21,13 @@
     <tag>HEAD</tag>
   </scm>
 
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://opensource.org/licenses/MIT</url>
+        </license>
+    </licenses>
+
```

Comment:
Actually related: I was adding a new file, so wanted to make sure the POM was declaring what license that would be under (which NetBeans takes into account when adding a default license header).